compiler/rustc_target/src/spec/nto_qnx_base.rs

```diff
@@ -0,0 +1,18 @@
+use crate::spec::{cvs, TargetOptions};
+
+pub fn opts() -> TargetOptions {
+    TargetOptions {
+        crt_static_respected: true,
+        dynamic_linking: true,
+        env: "nto71".into(),
+        executables: true,
+        families: cvs!["unix"],
+        has_rpath: true,
+        has_thread_local: false,
```

Comment:
Seeing how this option effects the thread_local macro and the thread_local test is failing; did you figure out if this should be on or off?

Reply:
I'm quite sure it should be disabled (from my current knowledge). The failing test case `src/test/ui/thread-local/tls.rs` is still failing for unknown reasons.
In case we find out that it is failing because of this parameter we can change it when supporting libstd (which implements the feature `#[thread_local]` which is tested by the named file).

Reply:
Adding the default destructor for TLS fixes this test case.  I messed around with getting the fast path for TLS support working but it involved a change to the llvm compiler and I was still having some issues.  Let's leave it disabled for now.  I'll revisit if we add __cxa_thread_atexit() in the future.